lang/en.json

```diff
@@ -887,6 +887,8 @@
         "TimeSinceLoadVariableUnitTitle": "Unit",
         "TimeSinceLoadVariableUnitDescription": "Select in which unit you want to retrieve the time.",
         "UrlParameterVariableNameTitle": "URL Parameter Name",
-        "UrlParameterVariableNameDescription": "For example when your page has URL parameters such as \"?lang=NZ\" and you want to get the value \"NZ\", then you need to enter \"lang\"."
+        "UrlParameterVariableNameDescription": "For example when your page has URL parameters such as \"?lang=NZ\" and you want to get the value \"NZ\", then you need to enter \"lang\".",
+        "UserInteractionTriggerName": "User Interaction",
+        "UserInteractionTriggerDescription": "Triggered when user interacts with the site."
```

Comment:
@JaJuMa Good to mention what all interactions can trigger this

Reply:
Eg: Triggered when any of the following event touch,move,scroll or mouse click is executed by the user on site.